digitalocean/database/resource_database_cluster.go

```diff
@@ -261,6 +279,18 @@ func resourceDigitalOceanDatabaseClusterCreate(ctx context.Context, d *schema.Re
 		opts.ProjectID = v.(string)
 	}
 
+	if v, ok := d.GetOk("backup_restore"); ok {
+		opts.BackupRestore = expandBackupRestore(v.([]interface{}))
+		originalDBName := opts.BackupRestore.DatabaseName
+		// waits for a a back up to be created before forking
+		err := waitForDatabaseBackups(client, d, originalDBName)
+		if err != nil {
+			d.SetId("")
+			return diag.Errorf("Error retrieving database cluster backups: %s", err)
+		}
```

Comment:
I wonder if this might be better as a test step? I'm not sure a real user would have this problem. So in most cases, it's an unneeded API call.

Reply:
Oh that's a good point. Highly unlikely someone will fork a newly created database. Just moved the logic to the test 👍